### PR TITLE
Reader: track the render of the follow by url button

### PR DESCRIPTION
**Pull request by samouri**
Adds render tracking for the follow by URL button in following manage.

Discussion: https://github.com/Automattic/wp-calypso/pull/14377#pullrequestreview-39791806

### CODE REVIEW

File: client/reader/following-manage/index.jsx [hunk 1]
```diff
@@ -137,6 +137,24 @@ class FollowingManage extends Component {
...
+			recordTrack( 'calypso_reader_following_manage_follow_by_url_render', {
+				url: siteUrl,
+			} );
+		}
+	};
+
+	shouldShowFollowByUrl = () => resemblesUrl( this.props.sitesQuery );
+	componentDidUpdate() {
+		if (
+			this.shouldShowFollowByUrl
```
Comment (blowery) [inline client/reader/following-manage/index.jsx]: need to call the function :)

Comment (blowery) [inline client/reader/following-manage/index.jsx]: also need to verify that the url changed from the last update and only report when it changes. and add reporting to componentDidMount, as componentDidUpdate doesn't fire during the initial mount.

**Comment by blowery** [comment]
nice, like the direction

File: client/reader/following-manage/index.jsx (hunk 1 above)
Comment (samouri) [inline client/reader/following-manage/index.jsx]: yea yea, the whole thing was blocked by mergin https://github.com/Automattic/wp-calypso/pull/14379

**Comment by samouri** [comment]
Just gave this a test run and functionality seemed good to me

File: client/reader/following-manage/index.jsx [hunk 2]
```diff
@@ -140,6 +142,25 @@ class FollowingManage extends Component {
...
+		const showingFollowByUrlButton = this.shouldShowFollowByUrl();
+
+		if ( siteUrl && showingFollowByUrlButton ) {
+			recordTrack( 'calypso_reader_following_manage_follow_by_url_render', {
+				url: siteUrl,
+			} );
+		}
+	};
+
+	shouldShowFollowByUrl = () => resemblesUrl( this.props.sitesQuery );
```
Comment (bluefuton) [inline client/reader/following-manage/index.jsx]: Didn't we stop using `resemblesUrl` somewhere along the line, and just trigger follow by URL based on a http:// or https:// prefix?

Comment (samouri) [inline client/reader/following-manage/index.jsx]: Not that I know of!

 I had a PR out with a "ready to merge" but it was considered an alternative to displaying both search results + the follow by url button.

https://github.com/Automattic/wp-calypso/pull/13957

Comment (bluefuton) [inline client/reader/following-manage/index.jsx]: 👍 👍

File: client/reader/following-manage/index.jsx [hunk 3]
```diff
@@ -198,7 +217,7 @@ class FollowingManage extends Component {
 						/>
 					</CompactCard>
 
-					{ isSitesQueryUrl &&
+					{ showFollowByUrl &&
```
Comment (bluefuton) [inline client/reader/following-manage/index.jsx]: Much clearer name 👍

**Comment by bluefuton** [comment]
Works well! One thing I noticed is that we're no longer sending along the correct `subscription_count` with the event - need to change it to look at Redux state I think:

screen shot 2017-05-25 at 14 46 50

**Comment by samouri** [comment]
> Works well! One thing I noticed is that we're no longer sending along the correct subscription_count with the event - need to change it to look at Redux state I think:

Going to need to update this within the `recordTrack` function of `reader/stats.js`.  I'll tackle this now but I'd rather it be a separate PR

**Comment by bluefuton** [comment]
In which case, 🚢 away!

**Pull request merged**